packages/docusaurus-theme-classic/src/theme/MDXComponents/Ul/index.tsx

```diff
@@ -22,5 +22,5 @@ function transformUlClassName(className?: string): string {
 }
 
 export default function MDXUl(props: Props): JSX.Element {
-  return <ul {...props} className={transformUlClassName(props.className)} />;
+  return <ul {...props} className={props.className ? transformUlClassName(props.className) : null} />;
```

Comment:
```suggestion
  return <ul {...props} className={props.className ? transformUlClassName(props.className) : undefined} />;
```

Fix CI

but not sure it passes linting, this is the problem with patch-x PRs 😅